### PR TITLE
test cases for testCompat.cxx file

### PR DESCRIPTION
**Pull request by shagunshaily**
made test cases for
-resipMin
-resipMax
-strncasecmp
-snprintf

### CODE REVIEW

**Comment by muonzoo** [comment]
Anyone object to merging this - it is still clean.

File: rutil/test/testCompat.cxx [hunk 1]
```diff
@@ -24,6 +24,141 @@ main()
...
+	   assert(resipMax(d11,d22) == d11);
+	   cerr<< "max(-5223372036854775807.99,-9223372036854775807.12): "<< resipMax(d11,d22)<< endl;
+	   //for same numbers
+	   assert(resipMax(3,3) == 3);
+	   cerr << "max(3,3 ): " << resipMax(3,3)<<endl;
+	   //for char type arguements
+	   assert(resipMax('a','b') == 'b');
+	   cerr<< "max('a','b'): "<<resipMax('a','b')<<endl;
+	   //test for string type arguements
+	   assert(resipMax("hello","jello") == "jello");
```
Comment (dpocock) [inline rutil/test/testCompat.cxx]: here you are comparing the pointers and not the strings.  Can you either remove these lines of code that compare strings, or represent the strings using a class such as std::string or resip::Data?

File: rutil/test/testCompat.cxx [hunk 2]
```diff
@@ -24,6 +24,141 @@ main()
...
+       assert(resipMin(d11,d22) == d22);
+	   cerr<< "min(-5223372036854775807.99,-9223372036854775807.12): "<< resipMin(d11,d22)<< endl;
+	   //for same numbers
+	   assert(resipMin(3,3) == 3);
+	   cerr << "min(3,3 ): " << resipMin(3,3)<< endl;
+	   //for char type arguements
+	   assert(resipMin('a','b') == 'a');
+	   cerr<< "min('a','b'): "<<resipMin('a','b')<<endl;
+	   //test for string type arguements
+	   assert(resipMin("hello","jello") == "hello");
```
Comment (dpocock) [inline rutil/test/testCompat.cxx]: see earlier comment about comparing pointers with resipMax

**Review by mannyad — CHANGES_REQUESTED** [review]
Pls

**Pull request closed without merging**